### PR TITLE
Profile page updates, fixes and changes

### PR DESCRIPTION
**Pull request by jrgarou**
### Profile Page
- Profile stylesheets
Created `stylesheets/pages/profile.less`: `/profile` page stylesheet;
Removed `account.less` and `wallet.less` imports.

- Edit profile dropdown
Created `stylesheets/components/dropdown/main.less` stylesheet;
Set `editProfileMenuOpen` class on `.main-app` to control the elements changes when `.main-dropdown` is open. There's no other dropdown in the player yet, so maybe I'll change the element to receive the class. Link to this event: https://github.com/Paratii-Video/paratii-player/blob/frontend/imports/ui/pages/profile/profile.js#L107-L125.

- editProfile/editPassword modal
Created `stylesheets/modal/profile.less` stylesheet;
New size to `.main-modal.wide`(editProfile modal).

### Other changes
- Some changes on  `internal-header` stylesheet to make dropdown work with buttons and fix `margin-right` of the title/search;
- Changed `main-modal-anim-NUMBER` class name to a "more global" name 🌐  : `anim-show-up-NUMBER`;
- mainFormInput component with inputId;
- Fix safari border-radius problem: https://github.com/Paratii-Video/paratii-player/blob/frontend/imports/ui/stylesheets/base/helpers.less#L59-L64;
- Set border on `.main-modal-label` instead `main-modal-input` to fix input over border on safari - and keep the autofill styles.

### CODE REVIEW

File: imports/ui/pages/profile/profile.html [hunk 1]
```diff
@@ -1,63 +1,51 @@
 <template name="profile">
   <div class="internals profile">
-    {{#if currentUser }} {{#isAuthorized}}
```
Comment (bent0b0x) [inline imports/ui/pages/profile/profile.html:3]: Just want to make sure: after these changes, is there still no scroll on the profile when embedded in twitter?

Comment (jrgarou) [reply imports/ui/pages/profile/profile.html:3]: Yes, @bent0b0x. Did you get something?

Comment (bent0b0x) [reply imports/ui/pages/profile/profile.html:3]: Nope, just confirming!

**Review by bent0b0x — APPROVED** [review]
Lots of stuff here but looks good to me

File: imports/ui/pages/profile/profile.html (hunk 1 above)
Comment (jrgarou) [reply imports/ui/pages/profile/profile.html:3]: Twitter embed is our reference now 😃

**Comment by jrgarou** [comment]
### Test changes

@bent0b0x I made some changes on profile test, besides rename classes selectors:

- Set `novalidate` attribute in the `#edit-profile-info-form` to avoid browser validation and pass [this test](https://github.com/Paratii-Video/paratii-player/blob/85b64b381753e0d99910bd5cd38629a48609351c/tests/profile.test.js#L644-L646):
https://github.com/Paratii-Video/paratii-player/blob/85b64b381753e0d99910bd5cd38629a48609351c/tests/profile.test.js#L644-L646
https://github.com/Paratii-Video/paratii-player/blob/d8163a94314df5e117b423a285b495fccb44b663/imports/ui/components/modals/editProfileInfo.html#L4
image

- Add `browser.waitForVisible()` before get elements values: https://github.com/Paratii-Video/paratii-player/commit/85b64b381753e0d99910bd5cd38629a48609351c


### Test error
The test stop at this line:
https://github.com/Paratii-Video/paratii-player/pull/372/commits/0de279a49e0e0350bcc361c347c9f9dcbe8bf8d1#diff-982d57c2cb605d32b4ba6976e9b9871bL648 
If we removed it the test passes.

**Comment by jellegerbrandy** [comment]
Yes, thanks guys!

**Pull request merged**